sci-libs/gdal/gdal1-1.11.5.recipe

```diff
@@ -116,26 +124,70 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
-defineDebugInfoPackage gdal1$secondaryArchSuffix \
-	$libDir/libgdal.so.$libVersion
+debugList=("$libDir"/libgdal.so.$libVersion)
+if [ -z "$commandSuffix" ]; then
+debugList+=(
+	"$commandBinDir"/gdal_contour \
+	"$commandBinDir"/gdal_grid \
+	"$commandBinDir"/gdal_rasterize \
+	"$commandBinDir"/gdal_translate \
+	"$commandBinDir"/gdaladdo \
+	"$commandBinDir"/gdalbuildvrt \
+	"$commandBinDir"/gdaldem \
+	"$commandBinDir"/gdalenhance \
+	"$commandBinDir"/gdalinfo \
+	"$commandBinDir"/gdallocationinfo \
+	"$commandBinDir"/gdalmanage \
+	"$commandBinDir"/gdalserver \
+	"$commandBinDir"/gdalsrsinfo \
+	"$commandBinDir"/gdaltindex \
+	"$commandBinDir"/gdaltransform \
+	"$commandBinDir"/gdalwarp \
+	"$commandBinDir"/nearblack \
+	"$commandBinDir"/ogr2ogr \
+	"$commandBinDir"/ogrinfo \
+	"$commandBinDir"/ogrlineref \
+	"$commandBinDir"/ogrtindex \
+	"$commandBinDir"/testepsg
+	)
+fi
+defineDebugInfoPackage gdal1$secondaryArchSuffix "${debugList[@]}"
 
 BUILD()
 {
 	expatDir=$portPackageLinksDir/lib~libexpat
 	autoreconf -fi
-	runConfigure ./configure --with-expat=$expatDir
+	runConfigure --omit-dirs "binDir dataDir" ./configure \
+		--bindir "$commandBinDir" \
+		--datadir="$dataDir"/gdal1 \
+		--with-expat=$expatDir
 	make $jobArgs
 }
 
 INSTALL()
 {
 	make install
 
+	if [ -n "$secondaryArchSuffix" ]; then
```

Comment:
Oops, I just found a latent bug which would affect users of x86_64 hybrid. That's my fault. Here's the fix:
```diff
-	if [ -n "$secondaryArchSuffix" ]; then
+	if [ -n "$secondaryArchSuffix" -a -z "$commandSuffix" ]; then
		mkdir -p "$binDir"
		mv "$commandBinDir"/gdal-config "$binDir"
	fi
```